src/swapd/runtime.rs

```diff
@@ -428,12 +435,19 @@ impl esb::Handler<ServiceBus> for Runtime {
         endpoints: &mut Endpoints,
         bus: ServiceBus,
         source: ServiceId,
-        request: Request,
+        request: BusMsg,
     ) -> Result<(), Self::Error> {
-        match bus {
-            ServiceBus::Msg => self.handle_rpc_msg(endpoints, source, request),
-            ServiceBus::Ctl => self.handle_rpc_ctl(endpoints, source, request),
-            _ => Err(Error::NotSupported(ServiceBus::Bridge, request.get_type())),
+        match (bus, request) {
+            // Peer-to-peer message bus
+            (ServiceBus::Msg, request) => self.handle_msg(endpoints, source, request),
+            // Control bus for internal command
+            (ServiceBus::Ctl, request) => self.handle_ctl(endpoints, source, request),
+            // User issued command RPC bus, only accept BusMsg::Rpc
+            (ServiceBus::Rpc, BusMsg::Rpc(req)) => self.handle_rpc(endpoints, source, req),
```

Comment:
nit: see comments comment elsewhere

Reply:
Done in [f28c0b2](https://github.com/farcaster-project/farcaster-node/pull/707/commits/f28c0b26a8a366ff8ad73e6285c37ae985c9a76c)